src/handlebars/users.handlebars

```diff
@@ -144,6 +144,10 @@
           <img src="https://static1.squarespace.com/static/5a78ccb7dc2b4a2c76787366/t/5a7a2be1652dead5a32a0f8b/1553085704781/?format=500w" alt="Utilus logo" />
         </div>
 
+        <div class="clients">
+          <img src="https://open-systems.com/wp-content/uploads/2019/07/open-systems-logo-horizontal.png" alt="Open Systems logo" />
```

Comment:
Do you have an svg logo available from a location such as wikimedia?

Reply:
updated